app/views/devise/passwords/edit.html.haml

```diff
@@ -4,20 +4,17 @@
   %span
 .entry-content.wpcf7
   = form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :put }) do |f|
-    - if resource.errors.any?
-      .wpcf7-response-output
-        - resource.errors.full_messages.each do |msg|
-          = msg
+
+    != model_errors_helper(resource)
+    %br/
```

Comment:
Done. See below for an example of this change in other forms. 

An unrelated note: The the text at the very top is not styled, and not seperated nicely from the text following it. Is this intentional or should we enter a bug?

<img width="1111" alt="membershipapplication" src="https://user-images.githubusercontent.com/19589814/28489726-e49342e2-6ec9-11e7-84e4-0c39d06ac0cb.png">